docs/integrations/builtin/credentials/shopify.md

```diff
@@ -15,12 +15,31 @@ You can use these credentials to authenticate the following nodes with Shopify.
 
 Create a [Shopify](https://shopify.com/){:target=_blank .external-link} account.
 
+Then [create a custom app](https://help.shopify.com/en/manual/apps/app-types/custom-apps){:target=_blank .external-link} to authenticate to.
+
 ## Supported authentication methods
 
-* Access Token
-* OAuth2
-* API Key
+* [Access Token](https://shopify.dev/docs/apps/auth/access-token-types/admin-app-access-tokens){:target=_blank .external-link}. Requires:
+    - Shop Subdomain (remove `.myshopify.com`)
+    - Access Token: In Shopify UI, the **Admin API Access Token**
+    - APP Secret Key: In Shopify UI, the **API Secret Key**
+* [OAuth2](https://shopify.dev/docs/apps/auth/get-access-tokens/token-exchange){:target=_blank .external-link}. Requires:
+    - Client ID: In Shopify UI, the **API Key**
+    - Client Secret: In Shopify UI, the **API Secret Key**
+    - Shop Subdomain (remove `.myshopify.com`)
+* API Key. Requires:
+    - API Key
+    - Password
+    - Shop Subdomain (remove `.myshopify.com`)
 
 ## Related resources
 
 Refer to [Shopify's authentication documentation](https://shopify.dev/docs/apps/auth){:target=_blank .external-link} for more information about the service.
+
+<!-- If this is a credential-only node, add a link to the node page on n8n's website. For example: https://n8n.io/integrations/356-gmail/ -->
+This is a credential-only node. Refer to [Shopify](/integrations/builtin/app-nodes/n8n-nodes-base.shopify/) and [Shopify Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.shopifytrigger/) to learn more. View [example Shopify workflows and related content](https://n8n.io/integrations/shopify/){:target=_blank .external-link} and [example Shopify Trigger workflows and related content](https://n8n.io/integrations/shopify-trigger/){:target=_blank .external-link} on n8n's website.
```

Comment:
This isn't a credential-only node. Credential only nodes are ones where we have creds but no app or trigger node.